src/index.html

```diff
@@ -6,14 +6,463 @@
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page-content">
+    <header
+      class="header"
+      id="home"
+    >
+      <nav class="header__navigation navigation">
+        <a
+          href="#menu"
+          class="navigation__item"
+        >
+          <img
+            src="./images/icons/menu-icon.svg"
+            alt="menu-icon"
+            class="navigation__icon navigation__icon--menu"
+          />
+        </a>
+
+        <a
+          href="#home"
+          class="navigation__item"
+        >
+          <img
+            src="./images/icons/logo-icon.svg"
+            alt="logo-icon"
+            class="navigation__icon"
+          />
+        </a>
+
+        <a
+          href="#shop"
+          class="navigation__item"
+        >
+          <img
+            src="./images/icons/shop-icon.svg"
+            alt="shop-icon"
+            class="navigation__icon"
+          />
+        </a>
+      </nav>
+
+      <img
+        src="./images/header-picture.jpg"
+        alt="header-picture"
+        class="header__picture"
+      />
+
+      <h1 class="header__title">Brand of eco-cosmetics and magic candles</h1>
+
+      <p class="header__text">20% of sales at a wild bird shelter</p>
+
+      <a
+        href="#shop"
+        class="button--type1 header__button--type1"
+      >
+        shop
+      </a>
+    </header>
+
+    <div
+      class="menu"
+      id="menu"
+    >
+      <a
+        href="#home"
+        class="menu__close-button"
+      ></a>
+      <ul class="menu__nav">
+        <li>
+          <a
+            href="#shop"
+            class="menu__link"
+          >
+            Shop
+          </a>
+        </li>
+        <li>
+          <a
+            href="#contact-us"
+            class="menu__link"
+          >
+            Contact
+          </a>
+        </li>
+        <li>
+          <a
+            href="#about"
+            class="menu__link"
+          >
+            About
+          </a>
+        </li>
+      </ul>
+    </div>
+
+    <main class="main">
+      <h2 class="main__text">
+        if you choose the gifts of nature and care about
+        <br />
+        it’s - scroll down
+      </h2>
+
+      <section
+        class="section"
+        id="about"
+      >
+        <h3 class="section__title">about our brand</h3>
+
+        <p class="section__text">
+          Our brand cosmetics are made from natural ingredients and we are
+          promoting them to the masses. Shampoo, hair conditioner, soap - in
+          recycled paper. Creams, oils, washing products in glass containers
+          with recycled paper labels. No synthetics and microplastics in
+          cosmetics. And the packaging from it can be returned for recycling or
+          reused.
+        </p>
+
+        <a
+          href="#address"
+          class="button--type2 section__button--type2"
```

Comment:
same problem with BEM (https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#using-a-modifier-without-the-belonging-class)